conf/model/torch.yaml

```diff
@@ -1,25 +1,26 @@
 model:
   name: pytorch
   build_params:
-    num_units: 50
-    dropout: 0.5
-    num_layers: 10
-    device: cpu
-    batch_size: 128
-    num_epochs: 10
-    scale_data: True
+    - network_class: MVRegressor
```

Comment:
please don't add the - here. they'll break the `build_model` functions.

Reply:
That actually worked for training the torch model. But, I switched back to the previous torch config.